### PR TITLE
Update react deps

### PR DESCRIPTION
**Pull request by jamesbhobbs**
**Overall change:** Update react deps to what are in package lock in [simorgh](https://github.com/bbc/simorgh)

**Code changes:**

- Just updates react deps

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

Very unlikely to break anything, I would just do a smoke test style regression

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -103,10 +103,10 @@
     "lint-staged": "^9.2.1",
     "prettier": "^1.18.2",
     "prop-types": "^15.7.2",
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
+    "react": "^16.9.0",
+    "react-dom": "^16.9.0",
     "react-helmet": "^5.2.0",
-    "react-test-renderer": "^16.8.6",
+    "react-test-renderer": "^16.9.0",
```
Comment (jamesbhobbs) [inline package.json:111]:
```suggestion
    "react-test-renderer": "^16.9.0", 
```

File: package.json [hunk 2]
```diff
@@ -103,10 +103,10 @@
...
```
Comment (jamesbhobbs) [inline package.json]:
```suggestion
    "react-test-renderer": "^16.9.0",
```

**Review by amywalkerdev — APPROVED** [review]
image

File: package.json [hunk 3]
```diff
@@ -103,10 +103,10 @@
     "lint-staged": "^9.2.1",
     "prettier": "^1.18.2",
     "prop-types": "^15.7.2",
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
+    "react": "^16.9.0",
```
Comment (dr3) [inline package.json:108]: We need to increase the versions in the packages to match this too

**Review by dr3 — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (jamesbhobbs) [reply package.json:108]: Thanks, forgot about peer deps

Comment (jamesbhobbs) [reply package.json:108]: Done, let me know if it's good to do a patch update on.

**Review by dr3 — COMMENTED** [review]
Needs changelogs and package bumps everywhere, tbh I recomend just waiting for https://github.com/bbc/psammead/pull/1828 to be merged, it will be able to do this PR in like 4 clicks

**Comment by jamesbhobbs** [comment]
> ogs and package bumps everywhere, tbh I recomend just waiting for #1828 to be merged, it will

Happy with that suggestion

**Pull request closed without merging**